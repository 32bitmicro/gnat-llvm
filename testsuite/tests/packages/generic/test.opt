ALL XFAIL Support for generic packages is missing
