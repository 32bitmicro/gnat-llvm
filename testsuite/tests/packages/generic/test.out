Foo (-1) = -1
Foo (0) = 0
Foo (1) = 1
Foo (4) = 4
Foo (8) = 8
